Marlin/src/inc/Conditionals_post.h

```diff
@@ -384,17 +406,23 @@
   #define BED_USES_THERMISTOR
 #endif
 
-#if TEMP_SENSOR_CHAMBER <= -2
-  #error "MAX6675 / MAX31855 Thermocouples not supported for TEMP_SENSOR_CHAMBER"
+#if TEMP_SENSOR_CHAMBER == -4
+  #define CHAMBER_USES_AD8495
+#elif TEMP_SENSOR_CHAMBER == -3
+  #error "MAX31855 Thermocouples not supported for TEMP_SENSOR_CHAMBER"
+#elif TEMP_SENSOR_CHAMBER == -2
+  #error "MAX6675 Thermocouples not supported for TEMP_SENSOR_CHAMBER"
 #elif TEMP_SENSOR_CHAMBER == -1
   #define CHAMBER_USES_AD595
 #elif TEMP_SENSOR_CHAMBER > 0
   #define THERMISTORCHAMBER TEMP_SENSOR_CHAMBER
   #define CHAMBER_USES_THERMISTOR
 #endif
 
+#define HEATER_USES_AD8495 (ENABLED(HEATER_0_USES_AD8495) || ENABLED(HEATER_1_USES_AD8495) || ENABLED(HEATER_2_USES_AD8495) || ENABLED(HEATER_3_USES_AD8495) || ENABLED(HEATER_4_USES_AD8495))
+
 #ifdef __SAM3X8E__
-  #define HEATER_USES_AD595 (ENABLED(HEATER_0_USES_AD595) || ENABLED(HEATER_1_USES_AD595) || ENABLED(HEATER_2_USES_AD595) || ENABLED(HEATER_3_USES_AD595))
+  #define HEATER_USES_AD595 (ENABLED(HEATER_0_USES_AD595) || ENABLED(HEATER_1_USES_AD595) || ENABLED(HEATER_2_USES_AD595) || ENABLED(HEATER_3_USES_AD595) || ENABLED(HEATER_4_USES_AD595))
```

Comment:
@Bob-the-Kuhn, @p3p —
Do you know why this option is limited to the `__SAM3X8E__`?
Should it be the same for the AD8495?

Reply:
@p3p — How come this option is limited to the __SAM3X8E__? Should it be the same for the AD8495? And… Does it mean this thermocouple amplifier cannot be used with other MCUs?